### DISCUSSION TITLE
change license to MIT or BSD

### DISCUSSION THREAD

**Discussion by leipzig** [broadinstitute/seqr#2953] [Q&A]
Hello,
I am noticing that seqr is one of the few projects in the [Broad organization](https://github.com/broadinstitute) that is not MIT or BSD licensed, but uses GPL aka copyleft licensing. This may deter commercial or private organizations from developing seqr. Not sure if this is intentional or what it would take to change the license now but I didn't want to leave that stone unturned.

Thanks!

**Comment by hanars** [comment]
seqr is an older application than most other Broad applications so we picked our license before they did which is why we aren't necessarily aligned with them. While there is probably no harm in switching to a different license, we don't really have any compelling reason to do so, as we aren't particular interested in commercial use of seqr and have not run into any hesitation on the part of other research collaborators, who are our intended users and collaborators
